How-To-Setup-Internal-Package-Repository.md

```diff
@@ -325,17 +308,19 @@ Note the section above where you should insert the code to test your packages be
 
 ```powershell
   function ConvertTo-ChocoObject {
+
       [CmdletBinding()]
+

```

Comment:
I'm not sure why this blank line was added - can you remove it?

Reply:
Addressed